examples/complete/main.tf

```diff
@@ -123,10 +51,6 @@ module "ec2_complete" {
   cpu_core_count       = 2 # default 4
   cpu_threads_per_core = 1 # default 2
 
-  capacity_reservation_specification = {
```

Comment:
this is what the fix enables - users do not need to set this if they are not using capacity reservations